app/views/404.server.view.html

```diff
@@ -1,8 +1,12 @@
 {% extends 'layout.server.view.html' %}
 
 {% block content %}
-<h1>Page Not Found</h1>
-<pre>
-	{{url}} is not a valid path.
-</pre>
+<div class="row">
+	<div class="col-xs-12">
+		<div class="container">
+			<h1>Page Not Found</h1>
+			You have entered an invalid URL.
```

Comment:
Can we get a "back to home" button? I think most users will click that pretty quickly.